jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/annotations/OnWebSocketMessage.java

```diff
@@ -35,9 +35,8 @@
  * <li><code>public void methodName(Reader reader)</code></li>
  * <li><code>public void methodName({@link Session} session, Reader reader)</code></li>
  * </ol>
- * Note: that the {@link Reader} in this case will always use UTF-8 encoding/charset (this is dictated by the RFC 6455 spec for Text Messages. If you need to
- * use a non-UTF-8 encoding/charset, you are instructed to use the binary messaging techniques.
- * <p>
+ * <p>Note: that the {@link Reader} in this case will always use UTF-8 encoding/charset (this is dictated by the RFC 6455 spec for Text Messages. If you need to
+ * use a non-UTF-8 encoding/charset, you are instructed to use the binary messaging techniques.</p><br>
```

Comment:
Remove `<br>` at the end of line.

Reply:
Sure, but I added this because its harder to read without it because there's no gap between this note and the "Binary Message Versions" header.